### DISCUSSION TITLE
Info on why it's called DBeaver, and possible rename

### DISCUSSION THREAD

**Discussion by chriszrc** [dbeaver/dbeaver#11422] [General]
Question # 1: Why is it called DBeaver? Is this documented anywhere? I couldn't find any info on the website or in github. 

Question # 2: Has anyone already suggested that this project be renamed? It's hard to even recommend this product to colleagues without getting strange looks. It's a great product, and I've been using it for a while, and would love to recommend it further-

**Comment by p91paul** [comment]
IMHO dbeaver is a great name: it makes clear that the tool is about db and beavers are awesome anyway. Moreover it is common practice to name db admin tools after animals (looking at you, SQL Squirrel and Toad), so strange looks on your colleagues for never having heard of dbeaver instead :smile:.

**Comment by serge-rider** [comment]
Answering the first question: in 2010, it was a hobby project. Initially intention was to replace SQL Squirrel for my own purposes. So it became "Database Beaver". Mostly because naming software products after animal names was a common practice in database world. And because beavers are awesome, as was rightly observed above :)

**Comment by benyanke** [comment]
I agree - I'm not seeing the issue with the name `dbeaver` either. It's not an especially odd name, to be honest, especially given the animal background in DBs. Anyways, FOSS often has odd names. MariaDB anyone? Almost identical, but with a random name instead of an animal.

**Comment by kennypellerin** [comment]
Great name.   However, I have people at work calling it DB-Eaver (emphasis on "Eaver"), even after seeing the picture.  I wonder if dBeaver vs DBeaver is in order.

**Comment by aisbergde** [comment]
Nevertheless, it is a good idea to describe that briefly in the documentation as well.

As a non-native English speaker, I have always wondered about the strange names for database tools and never tried to make sense of them. It would never have occurred to me to look for a translation. And so it was only today that I learned where all these terrible names come from that are so hard to remember and hard to pronounce: Toad, Squirrel. They are names of animals. And they are just pomposity of English-speaking developers. I would never have thought of that.  Even with dbeaver, I would never have thought of looking for beaver.

**Comment by skyvondave** [comment]
Maybe non-US people don't understand why we get odd reactions or laughter from others when we recommend DBeaver... it's because in the US, "beaver" isn't just a great animal in nature (they can revitalize forests after all), it's a slang term for a certain female body part (I'll let you guess which), in a certain range of curation.

I think its name has caused some of my colleagues to not take it seriously or not want to use it, even though they can see what a great interface it is.

I see the popularity of Sequel Pro, and many I've worked with are stuck on it even though it seems to be unsupported software, but it does seem that many are hesitant to use DBeaver just because of its name. While I love the interface, it hard to get around the name sometimes (no innuendo intended).

**Comment by chris4mich** [comment]
Beavers are often recognized as "ecosystem engineers," so, in my perspective, the selection of this animal as the namesake for the tool doesn't seem arbitrary.
